app/src/main/java/org/mozilla/fenix/settings/address/view/AddressEditorView.kt

```diff
@@ -27,9 +30,11 @@ class AddressEditorView(
 ) {
 
     /**
-     * Binds the view.
+     * Binds the given [AddressEditorState] in the [AddressEditorFragment].
+     *
+     * @param
      */
-    fun bind() {
+    fun bind(state: AddressEditorState) {
```

Comment:
We currently pass the `address` as parameter into `AddressEditorView`. We discussed this briefly in https://github.com/mozilla-mobile/fenix/pull/25031#discussion_r866336823.